src/PowerShellEditorServices/Debugging/DebugService.cs

```diff
@@ -686,9 +735,14 @@ private bool AddToAutoVariables(PSVariable psvariable, string scope)
         private async Task FetchStackFrames()
         {
             PSCommand psCommand = new PSCommand();
-            psCommand.AddCommand("Get-PSCallStack");
 
-            var results = await this.powerShellContext.ExecuteCommand<CallStackFrame>(psCommand);
+            // This glorious hack ensures that Get-PSCallStack returns a list of CallStackFrame
+            // objects (or "deserialized" CallStackFrames) when attached to a runspace in another
+            // process.  Without the intermediate variable Get-PSCallStack inexplicably returns
+            // an array of strings containing the formatted output of the CallStackFrame list.
+            psCommand.AddScript("$global:__psEditorServices_CallStack = Get-PSCallStack; $global:__psEditorServices_CallStack");
+
```

Comment:
I think I create variables like this for the hit count impl. We should probably create a string const somewhere like `PsesGlobalVariableNamePrefix = "__psEditorServices_";`